components/breadcrumb/stories/arg-types.js

```diff
@@ -0,0 +1,15 @@
+export const argTypes = {
+	connotation: {
+		control: {
+			type: 'select',
+			options: ['primary', 'cta', 'success', 'error', 'info', 'announcement'],
+		}
+	},
+	disabled: {
+		control: {
+			type: 'inline-radio',
+			options: { 'true': '', 'false': undefined }
+		}
+	},
```

Comment:
relevant?

Reply:
Nope. Can remove I guess. We don't have `disabled` breadcrumbs ;)